coalib/collecting/Collectors.py

```diff
@@ -350,6 +352,18 @@ def get_all_bears_names():
     return [bear.name for bear in get_all_bears()]
 
 
+def get_all_languages(include_unknown=False):
+    """
+    Get a ``list`` of all languages supported by coala, including
+    ``Unknown`` language.
+    """
+    languages = [
+        key for key in definitions.__dict__ if not key.startswith('__')]
```

Comment:
with `[key for key in coalib.bearlib.languages.definitions.__dict__]`, I was getting things like `__file__`, `__spec__` and some other dunders so I added the if statement. If there's another more efficient way to do it, please let me know. :)

Reply:
isinstance

Reply:
That helped a lot :+1: